src/backer/backering.py

```diff
@@ -299,20 +299,20 @@ def on_post(self, req, rep):
         cr = httping.parseCesrHttpRequest(req=req)
         serder = serdering.SerderKERI(sad=cr.payload, kind=eventing.Serials.json)
 
-        backer_identifier = serder.ked["b"]
+        backer_identifiers = serder.ked["b"]
         # Confirm registry backer
         if TraitCodex.Backers not in serder.ked["c"]:
             raise falcon.HTTPBadRequest(falcon.HTTP_400)
 
         contains_valid_seal = False
-        for item in serder.ked["a"]:
+        for key, item in enumerate(serder.ked["a"]):
             if (
                 item["bi"]
-                and item["bi"] in backer_identifier
                 and item["d"] == REGISTRAR_SEAL_SAID
             ):
-                contains_valid_seal = True
-                break
+                if backer_identifiers[key] == item["bi"]:
+                    contains_valid_seal = True
```

Comment:
same comment re counting as other PR (apologies I should've noticed this sooner)